docs/rst/platforms/platforms.rst

```diff
@@ -14,6 +14,9 @@ Vulcanexus ROS 2 all-in-one tool set, is officially available in the platforms s
     * - :ref:`Galactic Gamble <notes_galactic_latest>`
       - amd64
       - Ubuntu Focal (20.04)
```

Comment:
If there is not going to be a Vulcanexus Galactic release, I think we should delete all related information to Vulcanexus Galactic.

Reply:
Vulcanexus Galactic should be available at the same time Humble will be.